src/root/components/navbar.js

```diff
@@ -46,6 +46,31 @@ function Navbar(props) {
   const loadOptionsWithDebouncing = React.useCallback((input, callback) => {
     window.clearTimeout(debounceTimeoutRef.current);
 
+    let ans = [];
+    if (input.length < 3) {
```

Comment:
@frozenhelium @batpad could you please suggest me another condition (instead of input.length < 3) that would be true when the next request is not fulfilled?